fire_atlas_editor/lib/store/actions/atlas_actions.dart

```diff
@@ -125,16 +127,17 @@ class SaveAction extends SlicesAction<FireAtlasState> {
 }
 
 class LoadAtlasAction extends AsyncSlicesAction<FireAtlasState> {
-  String id;
+  String path;
```

Comment:
I'm not sure what kind of state manager you are using, but should actions be immutable?

Reply:
It is a homemade library. I am gradually improving it, immutability, not only on the actions, but on the state as well, is something on the list for followups.

Reply:
I meant, if that is the desire, you could just make it final here (not sure how that interops with your lib that I don't know)

Reply:
It is totally desirable, today I faced a weird bug to track because of mutability.

I will improve that on the future and do this on a single refactor. I am improving this gradually